### DISCUSSION TITLE
Multiple Sublabels

### DISCUSSION THREAD

**Discussion by MisterEcks** [blakeblackshear/frigate#9868] [Ideas]
It would be useful to be able to attach multiple value pairs as sublabels against and event/object rather than just the single sublabel attribute.

**Comment by NickM-27** [comment]
You can do a comma separated list. What case would there be for multiple sub labels?

**Comment by MisterEcks** [comment]
dynamic value pairs as sublabels would ease updating sublabel attributes/values individually.  managing the split/parsing of the sublabel data in external tools, scripts, integrations can be cumbersome. If sublabel could be populated by value pairs, then it could be used either simply or extensibly via JSON to augment events. Instead of splitting a sublabel attribute to find out a comma separated value hasn't been set, you could instead check for the existence of the sublabel name/type or its value without upstream parsing.
